SQL/New_patches/2022-12-15-Drop_Flag_status.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE `flag` DROP COLUMN `Flag_status`;
```

Comment:
Since it's dropping a column that might have data and existing projects might not want to run it should go in cleanup patches (which need to be manually run after review) instead of new_patches (which get concatenated and into one big release patch)

Reply:
Okay I will drop it under the cleanup patches then!